### ISSUE TITLE
Document event attributes that cannot be set in a POST /events API call

### ISSUE THREAD

**Issue by hillaryfraley** [sensu/sensu-docs#3133]
### Description
Certain event attributes are set by the backend and cannot be specified by the user when making an API call via POST /events. The documentation should specify which attributes these are.

### Category

This is an issue with:

- [ ] errata (error in doc content)
- [ ] new doc (requires a completely new doc)
- [x] update (add missing or refresh existing content)
- [ ] improvement (improve docs content or format)

### Affected pages
https://docs.sensu.io/sensu-go/latest/api/events/#events-post-specification
https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-events/events/#events-specification

### Expected behavior
Docs list the event attributes that a user cannot specify in a POST /events API call

### Current behavior
The docs do not list this information.

### Corrections needed
Add this information

### Additional context
Previous discussion about event attributes; possibly relevant: https://sensu.slack.com/archives/C7UNLR0F3/p1618340845047200

**Comment by hillaryfraley** [comment]
Asked Eric about where to find this information: https://sensu.slack.com/archives/C7UNLR0F3/p1618348788063200

**Comment by hillaryfraley** [comment]
I've confirmed these attributes reset to `0` if specified in a POST /events call:
`last_seen`
`total_state_change`

edit: if I manually create passing events with an `executed` value specified, Sensu updates `last_ok` to match the `executed` value. This happens even if the `executed` value is `0`.

It seems the backend also resets the values for these attributes to accurate (or automatically determined) numbers:
`occurrences`
`occurrences_watermark`
`state`

If I set `status` to a nonsense number like `75`, Sensu does not update it. Sensu does update the `state` to failing.

The backend also resets these attributes to `""`:
`libc_type`
`vm_system`
`vm_role`
`cloud_provider`
`sensu_agent_version`
